### DISCUSSION TITLE
Should Testflight be updated when a release is published?

### DISCUSSION THREAD

**Discussion by mcx808** [thebaselab/codeapp#464] [General]
My Testflight is still on 1.3.0 despite 1.3.1 being released a week ago. I've reinstalled & re-enrolled but still on that version.

**Comment by luni-moon** [comment]
I switched back to stable from the actual App Store, but I usually will switch back to testing, once version number is greater than stable version number.

**Comment by bummoblizard** [comment]
It should be synced again soon. I didn't put the previous builds on TestFlight because they are pretty trivial updates.

**Reply by bummoblizard** [reply]
1.3.4 is now on TestFlight
